api.planx.uk/.env.test.example

```diff
@@ -41,3 +41,4 @@ UNIFORM_TOKEN_URL=👻
 UNIFORM_SUBMISSION_URL=👻
 
 SLACK_WEBHOOK_URL=👻
+FILE_API_KEY=👻
```

Comment:
This is already defined above on L17, grouped in the `# AWS infra` section rather than `# Integrations` fyi!